.vscode/launch.json

```diff
@@ -7,10 +7,15 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Debug Jest with current test file",
-      "program": "${workspaceFolder}/packages/jest-cli/bin/jest.js",
-      "args": ["--runInBand", "${file}"],
-      "runtimeArgs": ["-r", "flow-remove-types/register"]
+      "name": "Jest Current File",
+      "program": "${workspaceFolder}/node_modules/.bin/jest",
+      "args": ["${fileBasenameNoExtension}", "--config", "jest.config.js"],
```

Comment:
Do we need to set `--config` explicitly? It should be inferred automatically

Reply:
It probably can be skipped - the whole thing is just copy-pasted from [here](https://github.com/microsoft/vscode-recipes/tree/master/debugging-jest-tests#configure-launchjson-file-for-your-test-framework). If you want me to remove it - let me know.

Reply:
Yea let's do it please :)

Reply:
worth a PR to that repo as well?

Reply:
```suggestion
      "args": ["${fileBasenameNoExtension}"],
```